Dockerfile

```diff
@@ -1,19 +1,24 @@
-ARG GOLANG_BASE
-ARG RUNNER_BASE
-
-FROM ${GOLANG_BASE} as builder
+FROM docker.io/golang:1.18.2-alpine3.15@sha256:e6b729ae22a2f7b6afcc237f7b9da3a27151ecbdcd109f7ab63a42e52e750262 AS builder
 RUN mkdir /.cache && chown nobody:nogroup /.cache && touch -t 202101010000.00 /.cache
 
+ARG GRPC_HEALTH_PROBE_VERSION=v0.4.11
```

Comment:
Yes, I have checked the Renovate, and it has more features than the Dependabot. Especially it is useful for our use cases (pinned docker containers, etc). I don't mind using it, as long as it's free :D Would you be up for creating an equivalent configuration? (considering your experience with it) It would speed things up!